webpack/modules/aws-cdk-lib/aws-lambda-nodejs/lib/esbuild.js

```diff
@@ -0,0 +1,45 @@
+// Load the go wasm.
+require("./go-wasm");
```

Comment:
if you are not planning to actually modify this file, instead of copying it in the pre-build plugin, you can simply alias "go-wasm" to the original one inside node_modules with webpack and just call `require("go-wasm")` here.